demo/api/src/importer/entities/base-import-target.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { BaseEntity, Constructor, Entity, PrimaryKey, Property } from "@mikro-orm/core";
+import { Field, ID, ObjectType } from "@nestjs/graphql";
+import { v4 } from "uuid";
+
+export interface BaseImportInterface {
+    id: string;
+    createdAt?: Date;
+    updatedAt?: Date;
+    deletedAt?: Date;
```

Comment:
I guess this is to implement a soft-delete. Is this something we want for every import?